docs/faq/terminology.rst

```diff
@@ -0,0 +1,46 @@
+.. _Observation_Terminology:
+
+***********
+Terminology
+***********
+
+Scheduling
+==========
+
+Scheduling observations is a common task in astronomy, but there is a lack of
+formal terminology. Here we define the terminology as it will be used in astroplan
+for scheduling tasks. This is for consistency within the code and the documentation.
+
+The terms used are as follows:
+
+* **visit**: a period of time spent observing a single target
+* **observing block** (OB): the minimum schedulable block (tentative: a group of visits)
+* **priority**: number assigned to the **observing block** by the user that 
+  defines its precedence within the set of blocks to be scheduled. Should probably
+  also be on a 0->1 (0=no good, 1=good) scale, or rescaled within the scheduler
+* **rank**: like a **priority**, but defined by a time allocation committee (TAC) rather
+  than a user and applied to an entire proposal.
+* **constraint**: sets limits and can yield **scores** based on inputs. Currently
+  only boolean scores are implemented in the code.
+* **score**: the value returned by evaluating a **constraint**. Can be 
```

Comment:
I think you want this to be "a constraint or multiple constraints" - when discussing the scorekeeper, ther terminology " a cumulative score to the OB" is used, which means this word can also apply to an OB.  I think that makes sense and is just fine, but you should note in the definition that a score can be cumulative on an OB in addition to being created by a constraint